src/ui.rs

```diff
@@ -615,6 +616,46 @@ fn create_view<Item, Model, RowData>(
     view.append_column(&column);
     view.add_css_class("data-table");
 
+    if Model::HAS_TIMES {
+        let model = model.clone();
+        let factory = SignalListItemFactory::new();
+        factory.connect_setup(move |_, list_item| {
+            let label = Label::new(None);
+            list_item.set_child(Some(&label));
+        });
+        let bind = move |list_item: &ListItem| {
+            let row = list_item
+                .item()
+                .context("ListItem has no item")?
+                .downcast::<RowData>()
+                .or_else(|_| bail!("Item is not RowData"))?;
+            let label = list_item
+                .child()
+                .context("ListItem has no child widget")?
+                .downcast::<Label>()
+                .or_else(|_| bail!("Child widget is not a Label"))?;
+            match row.node() {
+                Ok(node_ref) => {
+                    let node = node_ref.borrow();
+                    let timestamp = model.timestamp(&node.item);
+                    label.set_text(&format!("{}.{:09}",
```

Comment:
Telling the label to render in a small monospace font makes it easier to read:

```suggestion
                    label.set_markup(&format!("<tt><small>{}.{:09}</small></tt>",
```

Result:

![image](https://github.com/greatscottgadgets/packetry/assets/1239874/db278be5-aa7f-496e-a2e4-ca9085eb59ba)

Reply:
Good call, that definitely looks better.